python/eos/tasks.py

```diff
@@ -455,7 +455,7 @@ def sample_nested(analysis_file:str, posterior:str, base_directory:str='./', bou
     """
     analysis = analysis_file.analysis(posterior)
     results = analysis.sample_nested(bound=bound, nlive=nlive, dlogz=dlogz, maxiter=maxiter)
-    samples = np.array(list(map(analysis._x_to_par, results.samples)))
+    samples = _np.array(list(map(analysis._x_to_par, results.samples)))
```

Comment:
Was this not supposed to be corrected with respect to the calls to ``_x_to_par``?